R/across.R

```diff
@@ -101,7 +104,10 @@ across <- function(cols = everything(), fns = NULL, names = NULL) {
   names_data <- names(data)
 
   # promote formulas
-  fns <- map(fns, as_function)
+  args <- list2(...)
+  fns <- map(fns, function(f) {
```

Comment:
```suggestion
  fns <- map(fns, function(fn) {
```

To use standard particle `fn` in backtraces.